rts/Sim/Features/FeatureHandler.cpp

```diff
@@ -203,6 +220,9 @@ void CFeatureHandler::Update()
 
 		updateFeatures.erase(iter, updateFeatures.end());
 	}
+	if ((gs->frameNum & 62) == 0) {
```

Comment:
Shouldn't it be 63? With 62 it will do it twice in a row.

Reply:
you're right, changed it

Reply:
Where does 62 or 63 come from? Should they be some byproduct of `GAME_SPEED` or `*_SLOWUPDATE_RATE` ?

Reply:
Nowhere really, the number does have to be 2^X-1 I think to actually work properly and not have randomish steps. First I set it to 62 because 2 secs felt right, but didn't really think much about this using a mask instead of a % like other places do.

Not sure what the best practice is for this, didn't want to do the recalculation too much since also I don't foresee features flying a lot, but who knows better be prepared XD.